charts/optimize-live/values.yaml

```diff
@@ -9,19 +9,19 @@ component:
     controller:
       repository: registry.stormforge.io/optimize-live/controller
       pullPolicy: IfNotPresent
-      tag: latest
+      tag: 0.0.11
     tsdb:
       repository: registry.stormforge.io/optimize-live/tsdb
       pullPolicy: IfNotPresent
-      tag: latest
+      tag: 0.0.5
```

Comment:
isn't it 0.0.6 ? https://github.com/gramLabs/hungry-hungry-hippo/releases/tag/v0.0.6